frontend/src/main/components/Chat/ChatDisplay.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import ChatMessageDisplay from 'main/components/Chat/ChatMessageDisplay';
+import { useBackend } from "main/utils/useBackend";
+
+// Props for storybook manual injection
+
+const ChatDisplay = ({ commonsId, messages: messagesProp, userCommons: userCommonsListProp }) => {
```

Comment:
Avoid renaming props if you don't need to. You can avoid shadowing by overwriting the variables below.

Reply:
Same as above